src/index.js

```diff
@@ -8,6 +8,7 @@ import { RUNTIME, BUNDLED, HELPERS } from './constants.js';
 const keywordHelpers = [ 'typeof', 'extends', 'instanceof' ];
 
 export default function babel ( options ) {
+	const originalOptions = options;
```

Comment:
actually u shouldnt make a copy here, what's left in `options` (after all those `delete`s etc) is exactly what should get passed to babel - you might look how I've refactored it on master, this should be clearer there